lib/CMakeLists.txt

```diff
@@ -74,13 +74,30 @@ endif()
 #############
 # End logging configuration
 #############
-           
+
+if(NOT BOOSTFILESYSTEM) 
+target_link_libraries(ocpp
+    PUBLIC
+        everest::timer
+        websocketpp::websocketpp
+        nlohmann_json_schema_validator
+    PRIVATE
+        OpenSSL::SSL
+        OpenSSL::Crypto
+        SQLite::SQLite3
+        Threads::Threads
+
+        nlohmann_json::nlohmann_json
+        date::date-tz
+)
+else()
 target_link_libraries(ocpp
     PUBLIC
         everest::timer
         websocketpp::websocketpp
         nlohmann_json_schema_validator
     PRIVATE
+  		Boost::filesystem
```

Comment:
Would it be possible to add another `target_link_libraries()` with just these lines? To avoid doubling the other lines.